pkg/app/command/config.go

```diff
@@ -0,0 +1,27 @@
+package command
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/viper"
+)
+
+type Config struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Config should have comment or be unexported